src/mock/network.py

```diff
@@ -0,0 +1,18 @@
+# pylint: disable=too-few-public-methods
+
+"""
+    Mock Network
+"""
+
+
+class objectracker(object):
+    """Mock object tracker"""
+    missingObjects = ()
```

Comment:
should be an empty dict, not an empty list

Reply:
Done